h/views/api/annotations.py

```diff
@@ -160,6 +160,7 @@ def delete(context, request):
         context.annotation,
         'delete')
 
+    # TODO: Track down why we don't return an HTTP 204 like other DELETEs
```

Comment:
Occam's razor says it is because there was nothing providing clear visibility of such inconsistencies 😛